app/services/hyrax/existing_classes_query_service.rb

```diff
@@ -0,0 +1,60 @@
+# app/services/hyrax/existing_classes_query_service.rb
```

Comment:
I see that you defined a custom query `find_distinct_internal_resources` but not that it is actually used. 

We shouldn't have to check what the adapter is... each adapter should support the query.

Reply:
oh darn, maybe i forgot to commit something 👀 I was totally using it but you're right i don't see it here